### PR TITLE
added patch for filter option in electron-unhandled

### PR DESCRIPTION
**Pull request by vinodf2f**

### CODE REVIEW

File: patches/electron-unhandled+3.0.2.patch [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+ 
+ 	/**
+ 	Log an error. This does the same as with caught unhandled errors.
+-
+ 	It will use the same options specified in the `unhandled()` call or the defaults.
+-
+ 	@param error - Error to log.
+ 	*/
+ 	logError(error: Error, options?: unhandled.LogErrorOptions): void;
+ };
```
Comment (ritz078) [inline patches/electron-unhandled+3.0.2.patch:79]: Why are these added here ?

Comment (vinodf2f) [reply patches/electron-unhandled+3.0.2.patch:79]: Added typescript type for `filter` in index.t.ds

**Comment by ritz078** [comment]
@vinodf2f Can you fix https://github.com/ritz078/moose/issues/74 in this one too ? You will have to pass that error in filter option.

**Review by ritz078 — APPROVED** [review]
Great job. Make sure you undo this change when your original PR on electron-unhandled is merged.
